DOCS-7058 administer community (#128)
mulesoft/docs-api-community-manager: modules/ROOT/pages/mulesoft-support.adoc

= Contact MuleSoft Support
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images/

> For assistance, guidance, professional services customizations, and enhancement requests, please reach out to your MuleSoft Customer Success representative.

Reviewer: How about...:
```suggestion
For professional services customizations, enhancement requests, and other assistance, contact your MuleSoft Customer Success representative.
```

Reviewer 2: agree

> To report issues or defects in Anypoint API Community Manager, please submit a support case through the MuleSoft support portal, available at https://help.mulesoft.com .

Reviewer: Tech docs do not use "please"; GLOBALLY omit
Remove space
```suggestion
To report issues or defects in Anypoint API Community Manager, submit a support case through the MuleSoft support portal, available at https://help.mulesoft.com.
```

Reviewer 2: agree

> The MuleSoft support team works closely with the Salesforce support team and will coordinate work on any support tickets involving Salesforce technology.

Reviewer: ```suggestion
The MuleSoft support team works closely with the Salesforce support team to coordinate work on any support tickets involving Salesforce technology.
```

Reviewer 2: agree

> Service level agreements (SLAs) for support are available at the MuleSoft https://www.mulesoft.com/legal/support-maintenance-terms[Product Support and Maintenance Terms] page.

Reviewer: ```suggestion
Service-level agreements (SLAs) for support are available at the MuleSoft https://www.mulesoft.com/legal/support-maintenance-terms[Product Support and Maintenance Terms] page.
```

Reviewer 2: agree
